state_machine/applications/system-check/tests/sun_sensor_test.py

```diff
@@ -0,0 +1,174 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Sun Sensor Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+
+
+def lux(sun_sensor_index):
+    """ Return the lux reading for a given sun sensor """
+    if sun_sensor_index == "-Y":
+        return cubesat._cubesat.sun_yn.lux
+    elif sun_sensor_index == "-Z":
+        return cubesat._cubesat.sun_zn.lux
+    elif sun_sensor_index == "-X":
+        return cubesat._cubesat.sun_xn.lux
+    elif sun_sensor_index == "+Y":
+        return cubesat._cubesat.sun_yp.lux
+    elif sun_sensor_index == "+Z":
+        return cubesat._cubesat.sun_zp.lux
+    elif sun_sensor_index == "+X":
+        return cubesat._cubesat.sun_xp.lux
+    else:
+        print(sun_sensor_index, "is not a defined sun sensor.")
```

Comment:
I would rather see this raise a `ValueError` than just print something so we get the full stacktrace.